scripts/exregional_run_fcst.sh

```diff
@@ -329,13 +329,24 @@ of the current run directory (run_dir), where
   run_dir = \"${run_dir}\"
 ..."
 
+BKTYPE=1    # use INPUT
+if [ -r ${CYCLE_DIR}/INPUT/fv_tracer.res.tile1.nc ]; then
+  BKTYPE=0  # use RESTART
+fi
+print_info_msg "$VERBOSE" "
+The forecast has BKTYPE $BKTYPE (1:cold start ; 2 cycling)"
+
```

Comment:
What is BKTYPE=0? It's not reflected in this comment.

Reply:
This information was wrong. changed to ((1:cold start ; 0 cycling))